core/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -289,6 +292,42 @@ object VacuumCommand extends VacuumCommandImpl with Serializable {
 
 trait VacuumCommandImpl extends DeltaCommand {
 
+  private val supportedFsForLogging = Seq(
```

Comment:
Hope there is a configuration in `DeltaSQLConf` that allow to extend this FileSystem List that is supported for logging.

Reply:
Why wouldn't the user just set `vacuum.logging.enabled` to true?

The list `supportedFsForLogging` is only checked if `vacuum.logging.enabled` is not set.

Suppose we have some new key, `vacuum.logging.additionalSupportedFileSystem`.
So the user would have to leave `vacuum.logging.enabled` unset, then we would check the current FileSystem against `supportedFsForLogging` + ``vacuum.logging.additionalSupportedFileSystem` ... and then decide to do the logging.

It seems that just setting `vacuum.logging.enabled` to true is easier, eh?

Reply:
Good point. This convinced me. Let's just add `hdfs` to the list and not add the config right now. We can add the flag in future if needed.